charts/public-service/README.md

```diff
@@ -0,0 +1,103 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  
+
+- [Public available Service Helm Chart](#public-available-service-helm-chart)
+  - [Chart Details](#chart-details)
+    - [Verification](#verification)
+    - [Installation](#installation)
+    - [Service specific mandatory environment variables](#service-specific-mandatory-environment-variables)
+    - [Example](#example)
+    - [Known issues](#known-issues)
+      - [1. After service upgrade install `ImagePullBackOff` error is reported](#1-after-service-upgrade-install-imagepullbackoff-error-is-reported)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Public available Service Helm Chart
+
+This is the Helm Chart for services which must be public available, like shops, payment integrations etc.
+
+
+* Kubernetes 1.8+ with beta APIs enabled
+
+## Chart Details
+
+This chart will do the following:
+
+* Deploy an instance of the service (_Payone_ or _Paypal_) as on all nodes of a K8s cluster.
+
+### Verification
+
+Lint the chart
+
+```
+$ helm lint
+```
+
+To test the generated k8s resources from the helm chart
+
+```
+$ helm upgrade --install --dry-run --debug my-new-chart  .
+```
+
+### Installation
+
+To install the chart with the release name `my-service` using a dedicated namespace(recommended):
+
+```
+$ helm upgrade --install --namespace my-shop-namespace my-service -f values.yaml .
+```
+
+The chart can be customized using the following configurable parameters:
+
+| Parameter | Description | Default |
+| --- | --- | --- |
+| `image.repository` | docker image repository | ***mandatory*** |
+| `image.tag` | docker image tag | ***mandatory*** |
+| `image.pullPolicy` | docker image pull policy | `IfNotPresent` |
+| `replicaCount` | number of desired pods | `1` |
+| `nonSensitiveEnvs` | non sensitive environment variables | {} |
+| `sensitiveEnvs` | sensitive environment variables | {} |
```

Comment:
Please document all the missing values: `service` (and its kv map), `portName`, `ingress` and it's kv map

Reply:
97ce1f3